docs/template-vm.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: linux-vm-${NAME}
+  spec:
+    template:
+      metadata:
```

Comment:
In the template, do we count that the OfflineVirtualMachine name will be derived from template automatically? If not, the name should be defined here.

Reply:
I think this relates to what @fabiand commented right above

Reply:
As discussed above - The NAME parameter should be used as the name for the OfflineVM